index.html

```diff
@@ -1,206 +1,277 @@
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+        </section>
+        <section class="skills">
+            <h1 class="skills-heading">Skills</h1>
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>Github</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+        </section>
+    </section>
 
-        <!--Languages, frameworks, and skills-->
-
-        <section class="skills-section">
-            <section class="languages">
-                <h1 class="languages-heading">Languages</h1>
-                <ul>
-                    <li>Javascript(ES5/ES6)</li>
-                    <li>Ruby</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-            </section>
-            <section class="frameworks">
-                <h1 class="frameworks-heading">Frameworks</h1>
-                <ul>
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capibara</li>
-                    <li>Selenium</li>
-                </ul>
-            </section>
-            <section class="skills">
-                <h1 class="skills-heading">Skills</h1>
-                <ul>
-                    <li>Codekit</li>
-                    <li>Codepen</li>
-                    <li>Github</li>
-                    <li>GitLAb</li>
-                    <li>Terminal</li>
-                </ul>
-            </section>
+    <!--Form-->
+
+    <section class="contact-form">
+        <img alt="Logo" class="geometry" src="./images/image_geometry_1.png">
+        <h2 class="contact-heading">Get started</h2>
+        <p class="contact-paragraph">Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt
+            ut labore et dolore magna.</p>
+        <form class="form" action="https://formspree.io/f/mjvlwyao" method="POST">
+            <div class="input-box">
+                <label for="name">Your Name</label>
+                <input id="name" type="text" name="name" placeholder="Anastasia" required maxlength="30">
+            </div>
+            <div class="input-box">
+                <label for="email">Your Email</label>
+                <input id="email" type="email" name="email" required placeholder="emily@emily.com">
+            </div>
+            <textarea id="message" name="message" placeholder="Write your message here" required
+                maxlength="500"></textarea>
+            <button class="form-button" type="submit">Start Collaboration</button>
+        </form>
+    </section>
+
+    <!--Socials-->
+
+    <section class="socials">
+        <section class="resume">
+            <h2 class="get-resume">Get my resume</h2>
```

Comment:
- [ ] I really like the work that you did on adding the socials section at the end of the page but I suggest that you delete the socials section and put icons inside of the footer.